### PR TITLE
Cumulus 3836 fix clean executions connection

### PR DESCRIPTION
**Pull request by etcart**
**Summary:** Summary of changes

Addresses [CUMULUS-3836: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3836)

## Changes

* cleanExecutions.tf correctly passes through ES_HOST and lambda security group configuration

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [x] Integration tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -160,5 +160,8 @@
     "webpack-cli": "^4.9.0",
     "xml2js": "0.5.0"
   },
+  "overrides": {
```
Comment (Jkovarik) [inline package.json:163]: This is the vuln fix that @jennyhliu  noted yesterday?

**Review by Jkovarik — APPROVED** [review]
Approved based on pairing on 8 Aug

File: package.json (hunk 1 above)
Comment (jennyhliu) [reply package.json:163]: No. I added semver to audit-ci.json, and created a ticket

Comment (etcart) [reply package.json:163]: overrides Lerna's semver version and I ad-hoc tested it to make sure it doesn't interfere with Lerna's functioning

**Pull request merged**